### PR TITLE
CNV- 28046: Update runbooks

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.14

Issue: [CNV-28046](https://issues.redhat.com//browse/CNV-28046), [CNV-29425](https://issues.redhat.com/browse/CNV-29425)

Link to docs preview:
* [KubeVirtDeprecatedAPIsRequested](https://60965--docspreview.netlify.app/openshift-enterprise/latest/virt/support/virt-runbooks.html#virt-runbook-KubeVirtDeprecatedAPIsRequested)
* [KubeVirtNoAvailableNodesToRunVMs](https://60965--docspreview.netlify.app/openshift-enterprise/latest/virt/support/virt-runbooks.html#virt-runbook-KubeVirtNoAvailableNodesToRunVMs)

QE review: NA
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/virt-runbook-kubevirtdeprecatedapisrequested.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+:_content-type: REFERENCE
+[id="virt-runbook-KubeVirtDeprecatedAPIsRequested"]
+= KubeVirtDeprecatedAPIsRequested
+
+[discrete]
+[id="meaning-kubevirtdeprecatedapisrequested"]
+== Meaning
+
+This alert fires when a deprecated  {VirtProductName} API is used.
```
Comment (jldohmann) [inline modules/virt-runbook-kubevirtdeprecatedapisrequested.adoc:15]:
```suggestion
This alert fires when a deprecated {VirtProductName} API is used.
```

File: modules/virt-runbook-kubevirtnoavailablenodestorunvms.adoc [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+}
+{
+  "name": "shift-vwpsz-worker-gl5zl",
+  "kvm": "1k"
+}
+----
++
+Nodes with `"kvm": null` or `"kvm": 0` do not support virtualization extensions.
++
+Nodes with `"kvm": "1k"` do support virtualization extensions
```
Comment (jldohmann) [inline modules/virt-runbook-kubevirtnoavailablenodestorunvms.adoc:68]: seems like this would be better suited as code callouts

Comment (jldohmann) [inline modules/virt-runbook-kubevirtnoavailablenodestorunvms.adoc:68]:
```suggestion
Nodes with `"kvm": null` or `"kvm": 0` do not support virtualization extensions.
+
Nodes with `"kvm": "1k"` do support virtualization extensions.
```

**Review by jldohmann — COMMENTED** [review]
mostly LGTM, I left some comments below

File: modules/virt-runbook-kubevirtnoavailablenodestorunvms.adoc (hunk 2 above)
Comment (apinnick) [reply modules/virt-runbook-kubevirtnoavailablenodestorunvms.adoc:68]: Can't be done. These files are generated automatically from Markdown files and Markdown does not support code callouts.

**Pull request merged**